core/frontend/src/tile/TileAdmin.ts

```diff
@@ -160,15 +162,10 @@ export class TileAdmin {
    * @param props Options for customizing the behavior of the TileAdmin.
    * @returns the TileAdmin
    */
-  public static create(props?: TileAdmin.Props): TileAdmin {
-    return this.createForDeviceType(ProcessDetector.isMobileBrowser ? "mobile" : "non-mobile", props);
-  }
-
-  /** Strictly for tests.
-   * @internal
-   */
-  public static createForDeviceType(type: "mobile" | "non-mobile", props?: TileAdmin.Props): TileAdmin {
-    return new this("mobile" === type, props);
+  public static async create(props?: TileAdmin.Props): Promise<TileAdmin> {
+    const rpcConcurrency = IpcApp.isValid ? (await IpcApp.callIpcHost("queryConcurrency", "cpu")) : undefined;

```

Comment:
@kabentley at what point is it ok to call `callIpcHost`? From here in display-test-app I get a BackendError: Cannot read property 'queryConcurrency' of undefined.

```
callIpcChannel (IpcApp.ts:113)
await in callIpcChannel (async)
callIpcHost (IpcApp.ts:118)
create (TileAdmin.ts:166)
startup (IModelApp.ts:417)
startup (IpcApp.ts:123)
startup (NativeApp.ts:93)
startup (ElectronApp.ts:56)
startup (App.ts:175)
dtaFrontendMain (DisplayTestApp.ts:204)
```

Reply:
pushed fix

Reply:
fix is good.